core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleServicesDAO.scala

```diff
@@ -218,5 +200,13 @@ abstract class GoogleServicesDAO(groupsPrefix: String) extends ErrorReportable {
 }
 
 case class GoogleWorkspaceInfo(bucketName: String, policyGroupsByAccessLevel: Map[WorkspaceAccessLevel, WorkbenchEmail])
-case class ProjectTemplate(policies: Map[String, Seq[String]], services: Seq[String])
-final case class HammCromwellMetadata(bucketName: GcsBucketName, topicName: ProjectTopicName)
\ No newline at end of file
+case class ProjectTemplate(policies: Map[String, Seq[String]])
+final case class HammCromwellMetadata(bucketName: GcsBucketName, topicName: ProjectTopicName)
+
+case object ProjectTemplate {
+  def from(projectTemplateConfig: Config, requesterPaysRole: String): ProjectTemplate = {
```

Comment:
this is only used in test code so probably should move there. Also `requesterPaysRole` is unused

Reply:
it seems to be used in `WorkflowSubmissionActor`

Reply:
or do you mean it's unused in the `from` def

Reply:
yes on removing the unused var. it's not obvious to me where in test code it'd go -- might as well keep it near its class definition?